show_core.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+# Define project directories
+project_root="./"
+src_directory="./src"
+
+# Define files to show
+files_to_show=(
+  "src/main.ts"
+  "src/app/app.component.ts"
+  "src/app/app.config.ts"
+  "src/app/app.routes.ts"
+  "src/styles.scss"
+  "src/app/store/app.state.ts"
+  "src/app/store/user/user.actions.ts"
+  "src/app/store/user/user.selectors.ts"
+  "src/app/store/user/index.ts"
+)
```

Comment:
```suggestion
files_to_show=(
    "src/main.ts"
    "src/app/app.component.ts"
    "src/app/app.config.ts"
    "src/app/app.routes.ts"
    "src/styles.scss"
    "src/app/store/app.state.ts"
    "src/app/store/user/user.actions.ts"
    "src/app/store/user/user.selectors.ts"
    "src/app/store/user/index.ts"
)
```

Unify indentation across all code files. Four-space indentation is preferred.